portfolio/src/main/webapp/tsh.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    <meta charset="UTF-8">
+    <title>Quote</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head> 
+  <body>
+    <div id = "content">
+        <h1 class="header">The Secret History</h1>
+        <h3 class="subheader">by Donna Tartt (1992)</h3>
+        <p class="prompt">Want a cryptic quote from the Secret History?</p>
+        <div class="button">
+            <button onclick="addRandomGreeting()">Hmm...</button>
+        </div>
+        <div id="greeting-container"></div>
+        <div w3-include-HTML="navigation.html">Navigation</div>
+    </div>
+  </body>
+  <script> createNavigation() </script>
```

Comment:
Can you put this to the end of body tag as well? It's a common practice to put script in either head or body.

Reply:
Done.